.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -34,3 +34,5 @@ If applicable, add screenshots to help explain your problem.
 Add any other context about the problem here.
 e.g. code snippet to reproduce the issue.
 e.g. stack trace, memory dump, debugger log, and many etc.
+
```

Comment:
Is this not removing the template for bug reports? I think that one should stay?

Reply:
```general-inquiry.md``` is removed in this PR, so that users will not be given the option to create "general inquiry" when trying to create issue. The comment here is really trying to re-direct people to forum.